src/AccessibilityInsights.Core/Bases/A11yElement.cs

```diff
@@ -70,6 +85,9 @@ public string AcceleratorKey
             }
         }
 
+        /// <summary>
+        /// a text description of a keystroke which activates the element; usually a single key; only  valid within  a limited scope such as menus
```

Comment:
Extra spaces in this line

```suggestion
        /// a text description of a keystroke which activates the element; usually a single key; only valid within a limited scope such as menus
```